AdobeDocs/commerce-pwa-studio
# [Bugfix] Add placeholder content

## Description

Adds placeholder content to empty `index.md` files so the `build` script succeeds.

## Related Issue

Helps close PWA-1620

## Motivation and Context

Main currently does not build.

## How Has This Been Tested?

1. Run the `build` script (not the `dev` script)
2. Verify command succeeds and static assets are created
3. Run the `serve` script
4. Verify the serve command launches a working site

## Screenshots (if appropriate):

## Types of changes




## Checklist:
